app/containers/Avatar.js

```diff
@@ -15,27 +15,42 @@ const styles = StyleSheet.create({
 	},
 	avatarInitials: {
 		color: '#ffffff'
-	} });
+	}
+});
 
 class Avatar extends React.PureComponent {
 	render() {
-		const { text = '', size = 25, baseUrl = this.props.baseUrl,
-			borderRadius = 5, style, avatar } = this.props;
+		const { text = '', size = 25, baseUrl, borderRadius = 5, style, avatar } = this.props;
 		const { initials, color } = avatarInitialsAndColor(`${ text }`);
+
+		const iconContainerStyle = {
+			backgroundColor: color,
+			width: size,
+			height: size,
+			borderRadius
+		};
+
+		const avatarInitialsStyle = {
+			fontSize: size / 2
+		};
+
+		const avatarStyle = {
+			width: size,
+			height: size
+		};
+
+		const uri = avatar || `${ baseUrl }/avatar/${ text }`;
+		const image = (avatar || baseUrl) && (
```

Comment:
its wrong , we need the text behind the image. if the image is a svg type, the image cache will not render, in case of network error to.

Reply:
But the text is before the image, just in a variable to reduce code complexity.